src/store/index.ts

```diff
@@ -16,82 +17,88 @@ import { keyOfEntity, joinKeys, keyOfField } from '../helpers';
 import { query, write, writeFragment } from '../operations';
 
 export class Store {
-  records: EntitiesMap;
-  links: LinksMap;
+  records: Map<EntityField>;
```

Comment:
Nit: we can initialize here directly (will save quite a few bytes)

Reply:
It should transpile to the same output 😅